.travis-build.sh

```diff
@@ -73,6 +74,81 @@ if [ ! -z "$FORMATTING" ]; then
 		;;
 	esac
 fi
+
+# If we are pushing changes to the master branch,
+# open PR to syl20bnr/develop.spacemacs.org with Spacemacs
+# documentation exported as HTML and formatted with spacefmt
```

Comment:
@syl20bnr ok I'll make the bot open PRs to develop and master (based on which branch is updated)